htlcswitch/link.go

```diff
@@ -1875,7 +1875,7 @@ func (l *channelLink) updateCommitTx() error {
 
 	theirCommitSig, htlcSigs, err := l.channel.SignNextCommitment()
 	if err == lnwallet.ErrNoWindow {
-		l.tracef("revocation window exhausted, unable to send: %v, "+
+		l.debugf("revocation window exhausted, unable to send: %v, "+
```

Comment:
this could get pretty spammy on "hot" nodes, this can happen fairly regularly when htlcs are being queued